pwnagotchi/ui/hw/waveshare3.py

```diff
@@ -0,0 +1,48 @@
+import logging
+
+import pwnagotchi.ui.fonts as fonts
+from pwnagotchi.ui.hw.base import DisplayImpl
+
+
+class WaveshareV3(DisplayImpl):
+    def __init__(self, config):
+        super(WaveshareV3, self).__init__(config, 'waveshare_3')
+        self._display = None
+
+    def layout(self):
+        fonts.setup(10, 8, 10, 25, 25, 9)
```

Comment:
```suggestion
        fonts.setup(10, 8, 10, 35, 25, 9)
```

small face fix from https://old.reddit.com/r/pwnagotchi/comments/u4q18m/how_to_fix_the_small_face_issue_with_waveshare_v3/